gossip/blockproc/verwatcher/version_number.go

```diff
@@ -0,0 +1,38 @@
+package verwatcher
```

Comment:
misssing license header.

Reply:
Most files in this project do not have license headers. I think those should be added consistently in an independent PR.